verilog/formatting/tree_unwrapper.cc

```diff
@@ -636,6 +636,22 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeNode& node) {
         break;
       }
 
+    case NodeEnum::kIfClause:
+    case NodeEnum::kElseClause: {
+      bool gotseq = false;
+      for (const auto& itr : node.children()) {
+        if (static_cast<NodeEnum>(itr->Tag().tag) == NodeEnum::kSeqBlock)
```

Comment:
`.tag` could be either a node enumeration or leaf (token) enumeration depending on the type of the symbol, so to be safe, I would recommend explicitly checking whether the child is a tree-node first before using its tag.  

Also, do you need to scan every child?  or can you get away with examining specific children, positionally, using:

https://github.com/google/verible/blob/088be5950415a14f4036b366a826a7e2d9b4f679/common/text/tree_utils.h#L80

I think by construction in your updated verilog.y, you can check just the last child of these nodes for the statement block.  These are the kind of functions that best belong in the CST (statements?) library, like `GetIfClauseBody` and `GetElseClauseBody`.